assessments/projects/cni/docs/threat-modeling.md

```diff
@@ -0,0 +1,27 @@
+# CNI Threat Modeling
```

Comment:
Minor suggestion: consider renaming this to something like threat analysis, threat enumeration, lightweight threat assessment, etc. Presenting it as a threat model might be a bit misleading as that entails a comprehensive process and would be out of scope for the self-assessment (see: [OWASP Threat Modeling Process](https://owasp.org/www-community/Threat_Modeling_Process)).

Reply:
Thanks for the suggestion. I wasn't aware of the differences in terminology so I really appreciate the feedback!
I've renamed the references of threat modeling to lightweight threat assessment so it's clearer in describing the work we've done.

Reply:
No worries, looks good overall - thank you!